py-polars/polars/testing/parametric/strategies/dtype.py

```diff
@@ -89,6 +89,7 @@ def dtypes(
     *,
     allowed_dtypes: Collection[PolarsDataType] | None = None,
     excluded_dtypes: Sequence[PolarsDataType] | None = None,
+    allow_time_zones: bool = True,
```

Comment:
Feels like we need a helper class at this point. Will probably refactor in the future.